deploy/runtime/ui/PV Subhourly Clipping.json

```diff
@@ -0,0 +1,198 @@
+{
+    "Name": "PV Subhourly Clipping",
+    "Width": 1956.0,
+    "Height": 960.0,
+    "FormObjects": {
+        "GroupBox": {
+            "Visible": 1.0,
+            "ObjectProperties": {
+                "Name": {
+                    "Type": 5.0,
+                    "String": "object 4"
+                },
+                "X": {
+                    "Type": 3.0,
+                    "Integer": 18.0
+                },
+                "Y": {
+                    "Type": 3.0,
+                    "Integer": 21.0
+                },
+                "Width": {
+                    "Type": 3.0,
+                    "Integer": 1227.0
+                },
+                "Height": {
+                    "Type": 3.0,
+                    "Integer": 558.0
+                },
+                "Tool Tip": {
+                    "Type": 5.0,
+                    "String": ""
+                },
+                "Caption": {
+                    "Type": 5.0,
+                    "String": "PV Subhourly Clipping"
+                },
+                "Bold": {
+                    "Type": 2.0,
+                    "Boolean": 1.0
+                }
+            }
+        },
+        "DataMatrix": {
```

Comment:
Were we expecting this data matrix not to be a UI input anymore, since it's coded in SSC?

Reply:
This UI form is now no longer needed at all. Deleting.